Jenkinsfile.containers

```diff
@@ -0,0 +1,318 @@
+        MVN_BASE = "/usr/local/maven/bin/mvn --settings ${pwd()}/.ci/settings.xml"
+        MVN_COMMAND = "${MVN_BASE} --show-version --batch-mode --errors --fail-at-end -DdeployAtEnd=true "
+        M2_REPO = "${HOME}/.m2"
+        CI = credentials("app-jenkins")
+        SERVICE_SONAR_URL = credentials("service-sonar-java11-url")
+        SERVICE_NEXUS_URL = credentials("service-nexus-url")
+        SERVICE_REPO_SSHURL = credentials("repository-connection-string")
+        SERVICE_DOCKER_PULL_URL=credentials("SERVICE_DOCKER_PULL_URL")
+        SERVICE_REPOSITORY_URL=credentials("service-repository-url")
+        GITHUB_ACCOUNT_TOKEN = credentials("vitam-prg-token")
+
+    }
+
+
+        stages {
+
+
+
+            stage('Build and create docker images'){
+                steps {
+
+
+                    withCredentials([
+                    [$class: 'UsernamePasswordMultiBinding', credentialsId: 'app-jenkins',usernameVariable: 'CI_USR', passwordVariable: 'CI_PSW'],
+                    string(credentialsId: "service-nexus-url", variable: 'SERVICE_NEXUS_URL')
+                    ]) {
+
+
+                    sh 'mvn  --settings .ci/settings.xml --show-version --batch-mode --errors --fail-at-end -DinstallAtEnd=true -DdeployAtEnd=true package -Pvitam -DskipTests -DskipAllFrontend=true -DskipAllFrontendTests=true -Dlicense.skip=true  -Djib.to.auth.username=$CI_USR  -Djib.to.auth.password=$CI_PSW -pl "!cots/vitamui-nginx,!cots/vitamui-mongod,!cots/vitamui-logstash,!cots/vitamui-mongo-express" -Denv.SERVICE_NEXUS_URL=$SERVICE_NEXUS_URL -DsendCredentialsOverHttp=true'
+                    //sh '$MVN_COMMAND install jib:build  -Pvitam -Dmaven.test.skip=true -DskipTests=true -Djacoco.skip=true -Dlicense.skip -Denv.SERVICE_NEXUS_URL=$SERVICE_NEXUS_URL -DsendCredentialsOverHttp=true     -Djib.to.auth.username=$CI_USR  -Djib.to.auth.password=$CI_PSW -pl "!cots/vitamui-nginx,!cots/vitamui-mongod,!cots/vitamui-logstash,!cots/vitamui-mongo-express"'
+
+                    }
+
+
+
+                }
+
+            }
+        stage('Set scanner preriquities') {
+            steps {
+                // Install trivy
+                sh 'sudo curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/install.sh | sudo sh -s -- -b /usr/local/bin v0.47.0'
+                sh 'sudo curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/html.tpl > html.tpl'
+                    withCredentials([
+                    [$class: 'UsernamePasswordMultiBinding', credentialsId: 'app-jenkins',usernameVariable: 'CI_USR', passwordVariable: 'CI_PSW'],
+                    string(credentialsId: "service-nexus-url", variable: 'SERVICE_NEXUS_URL')
+                    ]) {
+                        sh 'echo $CI_PSW | docker login  pic-prod-nexus.vitam-factory:9080 -u $CI_USR --password-stdin'
+                    }
+            }
+        }
+
+/*        stage('Scan containers') {
+            steps {
+                // Scan all vuln levels
```

Comment:
Si on doit itérer unitairement sur tous les composants, ça serait mieux de faire une liste, d'ailleurs si on rajoute de nouveaux composants il ne faudra pas oublier de les lister ici.